packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -966,20 +982,50 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 
 	/**
 	 * {@inheritdoc IIntervalCollection.add}
+	 * @deprecated - call IntervalCollection.add without specifying an intervalType
```

Comment:
you need to deprecate the signature on the interface as well, not just the class. the class isn't exposed, so right now consumers won't see this deprecation unless they read the pnpm changeset.